packer/template.pkr.hcl

```diff
@@ -0,0 +1,53 @@
+packer {
+  required_version = ">= 1.7.0"
+  required_plugins {
+    amazon = {
+      version = ">= 1.0.0"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variable "aws_region" {
+  type    = string
+  default = "us-west-2"
+}
+
+source "amazon-ebs" "ubuntu" {
+  ami_name                = "sbtc-signer-image-{{timestamp}}"
+  instance_type           = "c6i.xlarge"
+  region                  = var.aws_region
+  source_ami_filter {
+    filters = {
+      name                = "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"
```

Comment:
We might want to consider using ubuntu 24.04 at this point, unless there is a good reason not to.